### DISCUSSION TITLE
Centos 6 Install protobuf-c error?

### DISCUSSION THREAD

**Discussion by syzh** [protobuf-c/protobuf-c#659] [Q&A]
config.status: executing libtool commands
  CXX    protoc-c/protoc_c_protoc_c-c_bytes_field.o
  CXX    protoc-c/protoc_c_protoc_c-c_enum.o
  CXX    protoc-c/protoc_c_protoc_c-c_enum_field.o
  CXX    protoc-c/protoc_c_protoc_c-c_extension.o
  CXX    protoc-c/protoc_c_protoc_c-c_field.o
  CXX    protoc-c/protoc_c_protoc_c-c_file.o
  CXX    protoc-c/protoc_c_protoc_c-c_generator.o
  CXX    protoc-c/protoc_c_protoc_c-c_helpers.o
  CXX    protoc-c/protoc_c_protoc_c-c_message.o
  CXX    protoc-c/protoc_c_protoc_c-c_message_field.o
  CXX    protoc-c/protoc_c_protoc_c-c_primitive_field.o
  CXX    protoc-c/protoc_c_protoc_c-c_service.o
  CXX    protoc-c/protoc_c_protoc_c-c_string_field.o
  CXX    protoc-c/protoc_c_protoc_c-main.o
  CXXLD  protoc-c/protoc-c
/usr/local/lib: file not recognized: Is a directory
collect2: ld returned 1 exit status
make: **\* [protoc-c/protoc-c] 错误 1

[root@localhost protobuf-c]$ protoc --version
libprotoc 2.5.0

**Comment by lipnitsk** [comment]
Are you building from a protobuf-c release tarball? If so, what is the output of `./configure`?
